### PR TITLE
add django 2.0 support

### PR DESCRIPTION
**Pull request by lynncyrin**
adds django 2.0 support ~by means of testing against [django@master](https://github.com/django/django)!~

### CODE REVIEW

**Comment by lynncyrin** [comment]
A quick search showed no recent usage of removal of this flag, so I assume it was depreciated some time ago.

**Comment by lynncyrin** [comment]
In case anyone else needs this in the meantime, the end user fix looks like this https://github.com/project-callisto/django-wizard-builder/pull/209/commits/a7f4a11c38826c93e5fbbfe34c1657c96f299274#diff-394e6b61ec4b04ef867603002928b3deR10

**Comment by MickaelBergem** [comment]
@lynncyrin I would be very interested in Django 2.0 support, what is the current status of your Pull Request? Do you need any help?

File: django_migration_testcase/base.py [hunk 1]
```diff
@@ -21,7 +21,7 @@ def func_wrapper(*args, **kwargs):
                 try:
                     func(*args, **kwargs)
                     transaction.savepoint_rollback(sp)
-                except:
+                except BaseException:
```
Comment (MickaelBergem) [inline django_migration_testcase/base.py]: 👍

File: tox.ini [hunk 2]
```diff
@@ -25,6 +26,7 @@ deps =
     django19: Django==1.9
     django110: Django==1.10
     django111: Django==1.11
+    djangodev: https://github.com/django/django/tarball/master
```
Comment (MickaelBergem) [inline tox.ini]: I think you can use `django20: Django==2.0`now that it has been released to pypi :)

**Comment by lynncyrin** [comment]
I think I just got distracted - I'll do some more work on this now

**Comment by lynncyrin** [comment]
https://docs.djangoproject.com/en/2.0/releases/2.0/#features-removed-in-2-0

> Field.rel and Field.remote_field.to are removed.

https://github.com/plumdog/django_migration_testcase/blob/8c0198b55f62b363a2c4f41918f5a6add514cb36/tests/test_app/tests.py#L177

Gotta figure out what to do about that

**Comment by lynncyrin** [comment]
image

fails are

- django14
- django15
- django16

**Comment by lynncyrin** [comment]
Not entirely sure whats causing the `django >= 1.6` fails, maybe someone else can look into it?

File: django_migration_testcase/base.py [hunk 3]
```diff
@@ -74,9 +74,10 @@ def run_reverse_migration(self):
         raise NotImplementedError()
 
     def migrate_kwargs(self):
-        return {'verbosity': 0,
-                'no_initial_data': True,
```
Comment (jrobichaud) [inline django_migration_testcase/base.py]: Anyone has any idea why `no_initial_data` was there? Compatibility for X version of django, South or something?

File: tox.ini [hunk 4]
```diff
@@ -9,22 +9,24 @@ envlist =
     py{27,34}-django17,
     py{27,35}-django1{8,9,10},
     py{27,36}-django111
+    py{36}-django2
```
Comment (jrobichaud) [inline tox.ini]: A comma is missing on the line above.

**Comment by MickaelBergem** [comment]
@plumdog I think we need help on this, maybe you could take a look?
I feel like we are very close to the end :)

**Comment by lynncyrin** [comment]
👀 looks pretty quite in here

someone else can handle fixing this particular thing, I think

**Comment by jrobichaud** [comment]
@plumdog do you need any help on maintaining this project? If you need help I could lend you a hand.

**Pull request closed without merging**